podcasts/New Search/Views/Results/SearchResultCell.swift

```diff
@@ -34,21 +44,25 @@ struct SearchResultCell: View {
                 HStack(spacing: 12) {
                     (episode?.podcastUuid ?? result?.uuid).map {
                         SearchEntryImage(uuid: $0, kind: result?.kind)
+                            .opacity(playedAlpha)
                     }
 
                     VStack(alignment: .leading, spacing: 2) {
                         if let episode {
                             Text(DateFormatHelper.sharedHelper.tinyLocalizedFormat(episode.publishedDate).localizedUppercase)
                                 .font(style: .footnote, weight: .bold)
                                 .foregroundColor(AppTheme.color(for: .primaryText02, theme: theme))
+                                .opacity(playedAlpha)
                             Text(episode.title)
                                 .font(style: .subheadline, weight: .medium)
                                 .foregroundColor(AppTheme.color(for: .primaryText01, theme: theme))
                                 .lineLimit(2)
+                                .opacity(playedAlpha)
                             Text(TimeFormatter.shared.multipleUnitFormattedShortTime(time: TimeInterval(episode.duration ?? 0)))
                                 .font(style: .caption, weight: .semibold)
                                 .foregroundColor(AppTheme.color(for: .primaryText02, theme: theme))
                                 .lineLimit(1)
+                                .opacity(playedAlpha)
```

Comment:
[nit] You can add this just one time after `.padding(.trailing, 8)` and the result will be the same.

Then we could change it to:

```swift
.opacity(played ? 0.5 : 1.0)
```

And on the constructor:

```swift
    init(episode: EpisodeSearchResult?, result: PodcastFolderSearchResult?, played: Bool = false) {
        self.episode = episode
        self.result = result
        self.played = episode != nil && played
    }
```

Removing the additional flag.